### ISSUE TITLE
Message sent even if tx failed

### ISSUE THREAD

**Issue by 0xFF0** [fr1t2/Quantum_Resistant_Ledger-Faucet#3]
Before sending a message on the blockchain, I think it good to verify if transaction was successful.

The json result from _relayTransferTxnBySlave()_ could be verified here before sending the message on the blockchain:

- https://github.com/fr1t2/QRL-Faucet/blob/master/script/payout.py#L121

- https://github.com/fr1t2/QRL-Faucet/blob/master/script/payout.py#L139

**Comment by fr1t2** [comment]
Agreed, validating the tx was successful before sending a confirmation message onto the chain makes sense. 

Will look to grab the result from `relayTransferTxnBySlave()` and validate the tx went through prior to sending message.

Optionally - adding a feature for collection of the failure messages to serve to the site owner either via email, or simply on a log in case something has broken...
